tools/util_windows.go

```diff
@@ -0,0 +1,163 @@
+// +build windows
+
+package tools
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"unsafe"
+
+	"github.com/rubyist/tracerx"
+	"golang.org/x/sys/windows"
```

Comment:
Do we need to add this to `go.mod`, or is it already brought in by the `golang.org/x/sys` package?

Reply:
We don't add entry to `go.mod`. It's part of [golang.org/x/sys](https://godoc.org/golang.org/x/sys).